result/arrow-parens.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-env es6*/
+
+// Bad
+a => {}
+
+// Good
+(a) => {}
+/*eslint-env es6*/
+
+// Bad
+if (a => 2) {
+}
+
+// Good
+if (a >= 2) {
+}
+/*eslint-env es6*/
+
+// Bad
+(a) => {}
+
+// Good
+a => {}
+/*eslint arrow-parens: ["error", "always"]*/
+/*eslint-env es6*/
+
+a => {};
+a => a;
+a => {'\n'};
+a.then(foo => {});
+a.then(foo => a);
+a(foo => { if (true) {}; });
+/*eslint arrow-parens: ["error", "always"]*/
+/*eslint-env es6*/
+
+() => {};
+(a) => {};
+(a) => a;
+(a) => {'\n'}
+a.then((foo) => {});
+a.then((foo) => { if (true) {}; });
+/*eslint-env es6*/
+
+var a = 1;
+var b = 2;
+// ...
+if (a => b) {
+ console.log('bigger');
```

Comment:
__[development][eslint]__
* Error - Unexpected console statement. (no-console)
* Error - 'console' is not defined. (no-undef)